persistence/remote/src/test/java/org/infinispan/persistence/remote/RemoteStoreConfigUriTest.java

```diff
@@ -0,0 +1,77 @@
+package org.infinispan.persistence.remote;
+
+import static org.infinispan.server.hotrod.test.HotRodTestingUtil.hotRodCacheConfiguration;
+import static org.infinispan.test.TestingUtil.withCacheManager;
+import static org.testng.AssertJUnit.assertEquals;
+
+import org.infinispan.Cache;
+import org.infinispan.client.hotrod.test.HotRodClientTestingUtil;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.server.hotrod.HotRodServer;
+import org.infinispan.server.hotrod.test.HotRodTestingUtil;
+import org.infinispan.test.AbstractInfinispanTest;
+import org.infinispan.test.CacheManagerCallable;
+import org.infinispan.test.TestingUtil;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ * Simple test to sample how remote cache store is configured.
+ *
+ * @author Mircea.Markus@jboss.com
+ * @since 4.1
+ */
+@Test(testName = "persistence.remote.RemoteStoreConfigUriTest", groups = "functional")
+public class RemoteStoreConfigUriTest extends AbstractInfinispanTest {
```

Comment:
As this class is essentiall the same a `RemoteStoreConfigTest`, I think we should modify `RemoteStoreConfigTest` so that it takes the `CACHE_LOADER_CONFIG` & `STORE_CACHE_NAME` as parameters in the constructor, with a default constructor using the current values. `RemoteStoreConfigUriTest` can then simply extend this class we can then just have a default constructor calling `super()` with the new configuration args.

e.g.:

```java
public RemoteStoreConfigUriTest() {
    super("remote-cl-uri-config.xml", RemoteStoreConfigUriTest.class.getSimpleName());
}
```

I know I previously mentioned having two hosts in the URI, however on reflection I don't think this is necessary as this is tested elsewhere and it will make ^ much simpler with a single server.